charts/bulk-scan-processor/values.yaml

```diff
@@ -15,6 +15,8 @@ java:
     OCR_VALIDATION_URL_FINREM: http://finrem-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     SCAN_ENABLED: "true"
     SCAN_DELAY: "4000"
+    UPLOAD_TASK_ENABLED: "true"
+    UPLOAD_TASK_DELAY: "4000"
```

Comment:
may be on PR you want less delay ?

Reply:
true, haven't checked preview template ;)

Reply:
hm. there's nothing there. deletion only amended to run every 10 seconds. maybe i'll just add there a delay of 2s

Reply:
https://github.com/hmcts/cnp-flux-config/pull/3139 already in place, nice.